.github/workflows/release-artifacts.yml

```diff
@@ -108,3 +112,88 @@ jobs:
         with:
           bin: qdrant
           token: ${{ secrets.GITHUB_TOKEN }}
+
+  build-app-image:
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-20.04
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y gcc-multilib clang cmake protobuf-compiler libfuse2
+
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v2
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+
+      - name: Build release binary
+        run: cargo build --release
+
+      - name: Build AppImage
+        shell: bash
+        run: |
+          mkdir -p AppDir
+          cp docs/logo.svg qdrant.svg
+          cp target/release/qdrant .
+
+          cat << EOF > "qdrant.desktop"
+          [Desktop Entry]
+          Name=qdrant
+          GenericName=Vector Database
+          Comment=Advanced and High-Performant Vector Search Technology
+          TryExec=qdrant
+          Exec=qdrant %F
+          Terminal=true
+          Type=Application
+          Keywords=Database;
+          Icon=qdrant
+          Categories=Utility;
+          StartupNotify=false
+          EOF
```

Comment:
On second thought, can we remove all EOF entries here into separate files to clean up this build script?

Maybe we can move this into `pkg/appimage/qdrant.desktop` for example.

Reply:
Sure!

Reply:
Cleaned up the code, have a look once again!